music21/analysis/neoRiemannian.py

```diff
@@ -391,9 +423,158 @@ def hexatonicSystem(c):
     # pragma: no-cover
     raise LRPException('Odd pitch class that is not in 0 to 11!')
 
+# ------------------------------------------------------------------------------
+
+def chromaticMediants(c, transformation='UFM', simplifyEnharmonics=True, raiseException=True):
+    '''
+    Transforms a chord into the given chromatic mediant. Options:
+    UFM = Upper Flat Mediant;
+    USM = Upper Sharp Mediant;
+    LFM = Lower Flat Mediant ;
+    LSM = Lower Sharp Mediant;
+
+    Each of these transformations
+    is mode-preserving;
+    involves root motion by a third;
+    entails exactly one common-tone.
+
+    >>> Cmaj = chord.Chord('C5 E5 G5')
+    >>> UFMCmaj = analysis.neoRiemannian.chromaticMediants(Cmaj, transformation='UFM')
+    >>> [x.name for x in UFMCmaj.pitches]
+    ['E-', 'G', 'B-']
+    >>> USMCmaj = analysis.neoRiemannian.chromaticMediants(Cmaj, transformation='USM')
+    >>> [x.name for x in USMCmaj.pitches]
+    ['E', 'G#', 'B']
+    >>> LFMCmaj = analysis.neoRiemannian.chromaticMediants(Cmaj, transformation='LFM')
+    >>> [x.name for x in LFMCmaj.pitches]
+    ['A-', 'C', 'E-']
+    >>> LSMCmaj = analysis.neoRiemannian.chromaticMediants(Cmaj, transformation='LSM')
+    >>> [x.name for x in LSMCmaj.pitches]
+    ['A', 'C#', 'E']
+    '''
+
+    if raiseException is True:
+        options = ['UFM', 'USM', 'LFM', 'LSM']
+        if transformation not in options:
+            raise NeoR_Exception('Transformation must be one of %s' %options)
+
+    if c.forteClassTnI == '3-11': # if (c.isMajorTriad() or chord.isMinorTriad()):
+        if transformation == 'UFM':
+            newChord = c.transpose(3)
+        elif transformation == 'USM':
+            newChord = c.transpose(4)
+        elif transformation == 'LFM':
+            newChord = c.transpose(-4)
+        elif transformation == 'LSM':
+            newChord = c.transpose(-3)
+
+    outChord = _simplerEnharmonics(newChord)
+    outChord.quarterLength = c.quarterLength
```

Comment:
this line seems unnecessary.  transpose definitely does and _simplerEnharmonics SHOULD (untried) preserve all other attributes.